app/scripts/modules/core/src/pipeline/config/stages/awsCodeBuild/awsCodeBuildStage.ts

```diff
@@ -0,0 +1,15 @@
+import { ExecutionDetailsTasks, Registry } from '@spinnaker/core';
+
+import { AwsCodeBuildStageConfig } from './AwsCodeBuildStageConfig';
+import { AwsCodeBuildExecutionDetails } from './AwsCodeBuildExecutionDetails';
+import { validate } from './AwsCodeBuildValidator';
+
+Registry.pipeline.registerStage({
+  label: 'AWS CodeBuild',
+  description: 'Trigger a build in AWS CodeBuild',
+  key: 'awsCodeBuild',
+  producesArtifacts: true,
```

Comment:
How does the CodeBuild artifact get tied to a Spinnaker pipeline artifact?

Reply:
CodeBuild artifact cannot be tied to Spinnaker artifact for now. I will need to implement it in later versions. The basic idea I have now is configuring a Spinnaker artifact as an output artifact override so that CodeBuild can upload artifact to that location. Then that Spinnaker artifact could be used in the future stages.
A better solution would be creating a Spinnaker artifact after build finishes and bind the output artifact to that newly created Spinnaker artifact. Not sure if this is doable. I will have to further look into both options.

Reply:
This 'producesArtifacts' attribute might need to be false until artifact support is plumbed through